R/mobr_boxplots.R

```diff
@@ -64,11 +66,16 @@ calc_biodiv_single_group <- function(abund_vec, n_rare){
    if (length(abund_vec) > 0){
       out_vec["N"] <- sum(abund_vec)
       out_vec["S"] <- length(abund_vec)
-      if (!is.na(n_rare))
-         out_vec["S_rare"] <- rarefaction(abund_vec, method = "indiv", effort = n_rare)
-      out_vec["PIE"] <- calc_PIE(abund_vec)
-      out_vec["ENS_PIE"] <- 1/(1 - out_vec["PIE"])
       
+      inext1 <- iNEXT::iNEXT(abund_vec, q = 0, datatype = "abundance", size = n_rare, se = F)
+      out_vec["S_rare1"] <- inext1$iNextEst$qD[1] 
```

Comment:
Here is the paper on iNEXT 
[Hsieh et al. - 2016 - iNEXT an R package for rarefaction and extrapolat.pdf](https://github.com/MoBiodiv/mobr/files/1108056/Hsieh.et.al.-.2016.-.iNEXT.an.R.package.for.rarefaction.and.extrapolat.pdf)

On Anne Chao’s website there is also on online shinyApp using the package
http://chao.stat.nthu.edu.tw/wordpress/software_download/inext-online/